client/src/partials/fiscal/fiscal.manage.html

```diff
@@ -21,27 +17,31 @@
             </div>
           </div>
 
-          <!-- date editor component -->
-          <!-- TODO - this label should say "Start Date" -->
-          <bh-date-editor
-            id="start_date"
-            date-value="FiscalManageCtrl.fiscal.start_date"
+          <!-- date interval component  -->
+          <bh-date-interval
+            date-from="FiscalManageCtrl.fiscal.start_date"
+            date-to="FiscalManageCtrl.fiscal.end_date"
+            mode="year"
+            on-change="FiscalManageCtrl.numberOfMonths()"
             validation-trigger="FiscalForm.$submitted"
-            required="required">
-          </bh-date-editor>
+            required="true">
+          </bh-date-interval>
 
           <div class="form-group" ng-class="{ 'has-error' : FiscalForm.$submitted && FiscalForm.number_of_months.$invalid }">
             <label class="control-label">{{ "FORM.LABELS.NB_MONTH" | translate }}</label>
-            <input type="number" class="form-control" ng-change="FiscalManageCtrl.endDate()" max="12" min="1" name="number_of_months" ng-model="FiscalManageCtrl.fiscal.number_of_months" required>
+            <input type="number" class="form-control" ng-change="FiscalManageCtrl.numberOfMonths()" max="12" min="1" name="number_of_months" ng-model="FiscalManageCtrl.fiscal.number_of_months" required>
             <div class="help-block" ng-messages="FiscalForm.number_of_months.$error" ng-show="FiscalForm.$submitted">
               <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
             </div>
           </div>
 
-          <div class="form-group text-muted">
-            <i class="fa fa-calendar"></i>
-            {{ "FORM.LABELS.END_DATE" | translate}}:
-            <span ng-show="FiscalManageCtrl.fiscal.number_of_months && FiscalManageCtrl.fiscal.start_date">{{ FiscalManageCtrl.end_date | date }}</span>
+          <div class="form-group">
+            <label class="control-label">{{ "FORM.LABELS.PREVIOUS_FISCAL_YEAR" | translate }}</label>
+            <select class="form-control"
+              ng-model="FiscalManageCtrl.fiscal.previous_fiscal_year_id"
+              ng-options="fy.id as fy.hrLabel for fy in FiscalManageCtrl.previous_fiscal_year">
```

Comment:
I just landed a ton of `ui-select` code for accounts.  A good example is on the [accounts modal](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/partials/accounts/edit/accounts.edit.modal.html#L69).

This can wait for a later PR.